toolkit/scripts/analysis.mk

```diff
@@ -71,6 +75,17 @@ sodiff-repo: $(SODIFF_REPO_FILE)
 
 $(SODIFF_REPO_FILE):
 	echo $(SODIFF_REPO_SOURCES) | sed -E 's:([^ ]+[.]repo):$(SPECS_DIR)/mariner-repos/\1:g' | xargs cat > $(SODIFF_REPO_FILE)
+	awk '{if ($$0 ~ /enabled=0/) $$0="enabled=1"; print}' $(SODIFF_REPO_FILE) > $(SODIFF_REPO_FILE)_fixed
```

Comment:
What is `$$0` used for?

Reply:
in this case the $$0 refers to the line that awk is reading in from the file. When concatenating the repo files together I want to ensure that each repo being added is enabled